tempesta_fw/http.c

```diff
@@ -1211,10 +1235,11 @@ tfw_http_resp_gfsm(TfwHttpMsg *hmresp, struct sk_buff *skb, unsigned int off)
 	req = tfw_http_popreq(hmresp);
 	if (unlikely(!req)) {
 		TFW_INC_STAT_BH(serv.msgs_filtout);
+		tfw_http_conn_msg_free(hmresp);
 		return TFW_STOP;
```

Comment:
It appears that in fact this should be `TFW_BLOCK` as this is about a server connection and the paired request is missing.